scenes/game/local_game.gd

```diff
@@ -3524,6 +3665,9 @@ func do_remaining_effects(performing_player : Player, next_state):
 			var effect = active_strike.remaining_effect_list[0]
 			active_strike.remaining_effect_list = []
 			events += do_effect_if_condition_met(performing_player, effect['card_id'], effect, null)
+
+			if len(active_strike.remaining_effect_list) > 0 and game_state == Enums.GameState.GameState_PlayerDecision:
```

Comment:
any reason to check the list length here and not just always break in player decision?

Reply:
I forget if I was avoiding anything specific; seems to be fine to remove the length check